src/mgr/DaemonHealthMetricCollector.cc

```diff
@@ -37,15 +37,15 @@ class SlowOps final : public DaemonHealthMetricCollector {
     if (daemons.empty()) {
       return;
     }
-    static const char* fmt = "%1% slow ops, oldest one blocked for %2% sec";
-    check.summary = boost::str(boost::format(fmt) % value.n1 % value.n2);
+    static const char* fmt = "%1% slow ops, oldest one blocked for %2% sec, %3%";
     ostringstream ss;
     if (daemons.size() > 1) {
       ss << "daemons " << daemons << " have slow ops.";
```

Comment:
We probably need to ellipsize this list at ~10 items to avoid gratuitously long log lines

Reply:
The capped list would be an improvement on the current one-or-nothing behaviour (it was clearly overkill to limit the current output to showing one or zero daemon names)

The full list of up to 1000s of daemons is clearly useful at the point someone is grepping/scripting rather than reading the output, but if we put the full-detail stuff in with the human readable stuff we end up destroying the human readability.

Unbounded length stuff in the main cluster log also disrupts any system downstream of us (UIs, etc) who has to somehow present the output.  If log lines become multi-kilobyte things each, then a UI that pulls down the last few hundred log lines is all of a sudden doing a multi-megabyte download.

However, we need to find a place to put this... see bottom comment